### PR TITLE
Cleanup of lazy props around solr ref guide

### PR DESCRIPTION
**Pull request by dweiss**

### CODE REVIEW

File: gradle/documentation/render-javadoc.gradle [hunk 1]
```diff
@@ -210,9 +210,6 @@ class RenderJavadocTask extends DefaultTask {
   @Input
   def offlineLinks = [:]
 
-  @Input
-  def luceneDocUrl = "${-> project.luceneDocUrl }"
-
   @Input
   def solrDocUrl = "${-> project.solrDocUrl }"
```
Comment (uschindler) [inline gradle/documentation/render-javadoc.gradle:214]: I think we should do here the same like in Lucene to allow snapshots without any link (null).

Comment (dweiss) [reply gradle/documentation/render-javadoc.gradle:214]: Yup. I can't do all of it though. I've spent a good two hours cleaning up other cruft. ;)

Comment (dweiss) [reply gradle/documentation/render-javadoc.gradle]: Updated the PR, seems to work.

Comment (uschindler) [reply gradle/documentation/render-javadoc.gradle]: Looks great, 😊

**Comment by uschindler** [comment]
What's wrong with precommit?

**Pull request closed without merging**